protostuff-collectionschema/src/main/java/io/protostuff/CollectionSchema.java

```diff
@@ -308,6 +308,22 @@ public static MessageFactories getFactory(String name)
         {
             return MessageFactories.valueOf(name);
         }
+
+        /**
+         * Check whether the specific class name can be accepted by factory.
+         */
+        public static boolean accept(String name)
+        {
+            try
+            {
+                getFactory(name);
+                return true;
+            }
+            catch (IllegalArgumentException e)
+            {
+                return false;
```

Comment:
an expensive way to check if it exists ... if there's no other way, then this will probably do

Reply:
Indeed. Iterating the enums is probably heavy. How about caching it as static?
```
public static boolean accept(String name)
        {
            try
            {
                return MESSAGE_FACTORIES_NAMES.contains(name);
            }
            catch (IllegalArgumentException e)
            {
                return false;
            }
        }
    }
    
    static Set<String> MESSAGE_FACTORIES_NAMES;
    
    static 
    {
        MessageFactories[] messageFactories = MessageFactories.values();
        MESSAGE_FACTORIES_NAMES = new HashSet<String>(messageFactories.length);
        for (MessageFactories messageFactory : messageFactories) {
            MESSAGE_FACTORIES_NAMES.add(messageFactory.name());
        }
    }
```

Reply:
I have updated the PR by using the less expensive way. Please check.